dataset/dictionary/enemies.json5

```diff
@@ -959,52 +1119,69 @@
   {
     en: "Rifthound Whelp",
     ja: "獣域ウェルプ",
+    zh: "兽境幼兽",
     pronunciationJa: "じゅういきウェルプ",
+    variants: {
+      zh: [ "小流血狗" ],
+    },
     notes: "whelp はイヌ科の動物の子供を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Rockfond Rifthound Whelp",
     ja: "獣域ウェルプ・岩",
+    zh: "兽境幼兽·岩",
```

Comment:
This seems wrong translation.

![](https://user-images.githubusercontent.com/107226297/174476488-d7d1c2a1-f9e2-457b-8e6f-a2137ce5e111.png)